package.json

```diff
@@ -109,7 +110,7 @@
   "main": "build/electron/electron/index.js",
   "husky": {
     "hooks": {
-      "pre-commit": "npx tslint --fix 'src/**/*.ts' && npx git-clang-format && npx pretty-quick --staged --pattern '**/*.html'"
+      "pre-commit": "npm run lint:ts && npx git-clang-format && npx pretty-quick --staged --pattern '**/*.html'"
```

Comment:
Add the --fix here.

I'm not a big fan of this pre-commit by the way. We may want to rethink it (not for this PR)

Reply:
Done.

Totally agree, I really dislike commit hooks myself. Not only can they just be bypassed, I feel dev velocity is impacted by:

- lack of live-reloading
- lack of supported editor (debatable) 
- lack of consistent folder structure, templating, and snippets
- coordinating updates across repositories
- lack of automated testing to debug against (this includes storybook)
- slow CI
- lack of fully automated deployments